src/ws/login.js

```diff
@@ -406,19 +415,27 @@
             } else {
                 show_form(in_conversation);
                 id("login-error-message").textContent = msg;
-                id("error-group").style.display = "block";
+                id("error-group").classList.replace("group-hidden", "group-visible");
```

Comment:
This _works_, but I was thinking that the display is grid by default and that hidden just changes it to to none. Then when hidden is removed, it defaults back (instead of toggling between displays in the CSS).

The JavaScript shouldn't have to care what display is being used for the element; it should just say "hide this" when applicable. That's it.

Reply:
(So, to fix this, just have the display type set in the styles of the elements... not toggle between `display: grid` and `display: none`.)